int16 moveTLR
int16 moveTFB
bool MOVE_RIGHT 
bool MOVE_FORWARD
bool CATCH_OR_NOT
